B827EBFFFE705686.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "B827EBFFFE705686",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": 50.9306288,
+    "ref_longitude": 5.3349823,
+    "ref_altitude": 55,
+    "contact_email": "steven.b@me.com,
```

Comment:
There's a missing quote at the end of the line, making the JSON invalid. Please fix and we'll merge. Thx! :)